backend/common/Helpers.js

```diff
@@ -70,8 +72,10 @@ Helpers.prototype.executePingJob = function(databaseName, jobObj, isOneTimeTest)
         if (resp) {
             console.log('Successfully executed the job : ', jobObj.jobId);
             var resultID = crypto.randomBytes(10).toString('hex');
-            var tagsObj = { jobid: jobObj.jobId, resultID: resultID };
+            var tagsObj = { jobid: jobObj.jobId, resultID: resultID,executedTime:jobObj.currentDateTime };
```

Comment:
var tagsObj = { jobid: jobObj.jobId, resultID: resultID, executedTime: jobObj.currentDateTime };
Corrected as that
Keep space after , and :